modules/cudf/src/table.ts

```diff
@@ -53,7 +53,7 @@ interface TableConstructor {
    * @param memory A buffer holding Arrow table
    * @return The Arrow data as a Table and a list of column names.
    */
-  fromArrow(memory: Memory|DeviceBuffer): {names: string[], table: Table};
+  fromArrow(memory: MemoryData): {names: string[], table: Table};
```

Comment:
This might need to be:

```suggestion
  fromArrow(memory: DeviceBuffer|MemoryData): {names: string[], table: Table};
```